cmd/buildah/run.go

```diff
@@ -178,5 +180,31 @@ func runCmd(c *cobra.Command, args []string, iopts runInputOptions) error {
 		conditionallyAddHistory(builder, c, "%s %s", shell, strings.Join(args, " "))
 		return builder.Save()
 	}
+	// unlock if any locked files from this RUN statement
+	for _, path := range lockedTargets {
+		_, err := os.Stat(path)
+		if err != nil {
+			// Lockfile not found this might be a problem,
+			// since LockedTargets must contain list of all locked files
+			// don't break here since we need to unlock other files but
+			// log so user can take a look
+			logrus.Errorf("Lockfile %q was expected here, stat failed with %v", path, err)
```

Comment:
Is there anything a user could do at this point to correct this?  Or support for that matter?

Reply:
@TomSweeneyRedHat No user cannot do anything but I think this is worth logging cause following case is never expected to happen and if it happens it could be a symptom of a bigger problem in our fs locking library or anywhere above that.

Reply:
That's what I thought @flouthoc .  I was kind of waffling about whether this should be a debug or not, but probably better to keep it as an error.

Reply:
Errors should cause the app to exit, if this does not then this should be a warning.